kagiso_auth/templates/kagiso_auth/forgot_password.html

```diff
@@ -5,11 +5,14 @@
 
 {% block title %}Forgot Password{% endblock %}
 
-{% block uncached_content %}
+{% block content %}
 
-  <h1>Forgot Password</h1>
   <section class="form-main-wrapper password-form-container">
-    <form action="{% url 'forgot_password' %}" method="post" class="form-generic">
+  	<h1>Forgot Password</h1>
+    <form
+    	action="{% url 'forgot_password' %}"
+    	method="post"
+    	class="form-generic">
       {{ form.as_p }}
       <input type="submit" value="Reset Password" class="btn btn-primary btn-lg">
```

Comment:
Let's use our button classes: `btn-primary-lg`. You will need to create it on the CMS side - we only have a `.btn-primary-md`.